ArchitectureHooks.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <binaryninjaapi.h>
+#include "../../scc/RefCountObject.h"
+
+// This aims to be a safe expression as we cant be 100% sure what the underlying arch is doing
+// It is verified safe on x64 and aarch64 which this plugin currently exclusively targets
+constexpr uint32_t cfstrIntrinsicIndex = UINT32_MAX - 64;
+
```

Comment:
Seems that this file is using core naming conventions and that this should be `CFSTRIntrinsicIndex`.